Cargo.toml

```diff
@@ -17,7 +17,7 @@ default-members = [
 resolver = "2"
 
 [workspace.package]
-version = "0.7.1"
+version = "0.7.2"
```

Comment:
I think we should switch back to `0.8.0` because we have breaking changes in `main`.